### DISCUSSION TITLE
TP7 - Problema con incluir un ejercicio como libreria

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#196] [Preguntas y respuestas] [answered]
Hola buenas, estoy tratando de utilizar el ejercicio de suma_lenta con un #include como pide la consigna del tp7 pero cuando compilo me dice:

$> fatal error: Tp1ejercicio5.c: No such file or directory
    3 | #include <Tp1ejercicio5.c>

ya probe teniendo el ejercicio con ese nombre en la misma carpeta donde ejecuto el programa y luego probe crear una carpeta llamada Tp1ejercicio5 dentro de la carpeta donde ejecuto el programa y ninguna manera resulta, como deberia hacer?

**Comment by Nikzza** [comment, marked as answer]
Se utiliza 

```C
#include "Tp1ejercicio5.c"
```

**Reply by martinvilu** [reply]
https://github.com/orgs/INGCOM-UNRN-P1/discussions/182

Indicado por @Nikzza
